pyccl/halos/profiles.py

```diff
@@ -989,31 +1191,39 @@ def _integ_interp(self):
         from scipy.integrate import quad
 
         def integrand(x):
-            return self._form_factor(x)*x
+            return self._form_factor(x) * x
 
         q_arr = np.geomspace(self.qrange[0], self.qrange[1], self.nq)
         # We use the `weight` feature of quad to quickly estimate
         # the Fourier transform. We could use the existing FFTLog
         # framework, but this is a lot less of a kerfuffle.
-        f_arr = np.array([quad(integrand,
-                               a=1e-4, b=self.x_out,  # limits of integration
-                               weight="sin",  # fourier sine weight
-                               wvar=q)[0] / q
-                          for q in q_arr])
-        Fq = interp1d(np.log(q_arr), f_arr,
-                      fill_value="extrapolate",
-                      bounds_error=False)
+        f_arr = np.array(
+            [
+                quad(
+                    integrand,
+                    a=1e-4,
+                    b=self.x_out,  # limits of integration
+                    weight="sin",  # fourier sine weight
+                    wvar=q,
+                )[0]
+                / q
+                for q in q_arr
+            ]
+        )
+        Fq = interp1d(
+            np.log(q_arr), f_arr, fill_value="extrapolate", bounds_error=False
+        )
         return Fq
 
     def _norm(self, cosmo, M, a, mb):
         # Computes the normalisation factor of the GNFW profile.
         # Normalisation factor is given in units of eV/cm^3.
         # (Bolliet et al. 2017).
-        h70 = cosmo["h"]/0.7
-        C0 = 1.65*h70**2
-        CM = (h70*M*mb/3E14)**(2/3+self.alpha_P)   # M dependence
-        Cz = h_over_h0(cosmo, a)**(8/3)  # z dependence
-        P0_corr = self.P0 * h70**self.P0_hexp  # h-corrected P_0
+        h70 = cosmo["h"] / 0.7
+        C0 = 1.65 * h70 ** 2
+        CM = (h70 * M * mb / 3e14) ** (2 / 3 + self.alpha_P)  # M dependence
+        Cz = h_over_h0(cosmo, a) ** (8 / 3)  # z dependence
+        P0_corr = self.P0 * h70 ** self.P0_hexp  # h-corrected P_0
```

Comment:
Same thing here about the padding around the power operator. Reading the code, this might look like it meant to be a multiplication with a typo. Not padding directly refers to raising the base to a power.